### DISCUSSION TITLE
BIG Digital Temps (portrait) [3.5"]

### DISCUSSION THREAD

**Discussion by geekm0nkey** [mathoudebine/turing-smart-screen-python#296] [Themes]
Here is a large digital display for temps and cpu/gpu load. Very minimalistic. 

![screencap](https://github.com/mathoudebine/turing-smart-screen-python/assets/35175147/dff6e0ec-3760-4f85-af6a-b8647d1d2e1d)

**Download**
[BIG Digital (portait).zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12114288/BIG.Digital.portait.zip)

**Comment by nksoft** [comment]
I loved your theme so much but don't have a GPU on my laptop so had to change it to only show CPU. Hope that's ok!

![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/72239803/1dbbf74e-703d-4aff-ab70-71f3cfb5ae3a)

Download
[BIG Digital CPU (portrait).zip](https://github.com/mathoudebine/turing-smart-screen-python/files/12472780/BIG.Digital.CPU.portait.zip)

**Comment by geekm0nkey** [comment]
Great job. I do not mind at all, in fact, I encourage it. That is why on my releases I supply everyone with the source content so they can customize it to their own needs.

**Comment by Agudelo91** [comment]
I love it,
Could it be in landscape? 
Thanks :)

**Comment by RAMENPOTAFRAISE** [comment]
Hello, thanks for this theme, I rly love it! Could you might be add a mouse batterie in % on it?! 

Ty !

**Comment by alexzad** [comment]
best theme ever!

**Comment by DJRoby19** [comment]
You could do a horizontal version that would be great

**Reply by nexpid** [reply]
https://github.com/mathoudebine/turing-smart-screen-python/discussions/538
